.travis.yml

```diff
@@ -1,16 +1,13 @@
 language: python
 python:
     - "2.7"
+env:
+    - TOX_ENV=py27,flake8,pylint
 install:
     - pip install -U pip
     - pip install -r requirements.txt
     - pip install -r test-requirements.txt
```

Comment:
Can you make travis config in a similar way as skt has? We are doing the installation of skt there directly, without mentioning the requirements, to ensure it works, and also testing uninstalling.